sass/elements/tag.sass

```diff
@@ -8,6 +8,16 @@ $tag-delete-margin: 1px !default
   display: flex
   flex-wrap: wrap
   justify-content: flex-start
+  &.is-centered
+    justify-content: center
+    .tag
+      margin-right: 0.25rem
+      margin-left: 0.25rem
+  &.is-right
+    justify-content: flex-end
+    .tag
```

Comment:
You need to use `.tag:not(:last-child)`

Reply:
@jgthms thanks for taking the time to review this. I might have misunderstood, but if I use `.tag:not(:last-child)` in this block:

``` sass
&.is-right
    justify-content: flex-end
    .tag
      margin-left: 0.5rem
      margin-right: 0
```

It will cause the last child to not get the appropriate left margin - which will look like this:

<img width="346" alt="screen shot 2017-11-05 at 9 50 13 pm" src="https://user-images.githubusercontent.com/24803032/32414096-b33924ac-c273-11e7-9ea2-bb46e7f22034.png">

If I'm missing something let me know and I'm more than happy to fix it up - thanks for the great framework!

Reply:
Ah sorry, it should be this I believe:

```
&.is-right
  .tag
    &:not(:first-child)
      margin-left: 0.5rem
    &:not(:last-child)
      margin-right: 0
```

Reply:
@jgthms  yup - I get you. makes sense - more targeted. Thanks. I've updated that for you.